test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -501,19 +566,19 @@ def load_config(self, cfg):
         """
         Create a hook configuration file with the provided contents.
         """
-        fn = self.du.create_temp_file(body=cfg)
+        fn = self.du.create_temp_file(hostname=self.serverA, body=cfg)
         a = {'content-type': 'application/x-config',
              'content-encoding': 'default',
              'input-file': fn}
         self.server.manager(MGR_CMD_IMPORT, HOOK, a, self.hook_name)
-        os.remove(fn)
-        self.mom.log_match('pbs_cgroups.CF;copy hook-related ' +
-                           'file request received',
-                           max_attempts=5,
-                           starttime=self.server.ctime)
+        self.hostA.log_match('pbs_cgroups.CF;copy hook-related '
```

Comment:
I am afraid I did not get your comment. is this on the wrong line?

Reply:
create_temp_file() does not delete the temp file.  It can't.  It will close the file descriptor that mkstemp() opens for you.  It can't delete the file because the whole purpose of the function is to create you a file in which you can use.  If it deleted the file, it wouldn't be there for you to use.

Reply:
Ah sorry .. yes of course (I don't know what was I thinking). I will add it back.